package.json

```diff
@@ -6,33 +6,48 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
+    "build:deploy": "npm run build && npm run deploy:s3",
+    "client:deploy": "sls client deploy --no-config-change --no-policy-change --no-cors-change",
```

Comment:
It looks like ```deploy:s3``` and ```client:deploy```  execute the same commands and essentially duplicate each other.
I think that only one is needed and the second is redundant and can be removed. What do you think?

Reply:
Yes, `deploy:s3` is part of subtask of creating automated deploy on s3. Is it important to delete it?